hypothesis-python/src/hypothesis/strategies/_internal/attrs.py

```diff
@@ -140,11 +174,11 @@ def types_to_strategy(attrib, types):
 
     # Otherwise, try the `type` attribute as a fallback, and finally try
     # the type hints on a converter (desperate!) before giving up.
-    if is_a_type(getattr(attrib, "type", None)):
+    if (attrib_type := attrib.type) is not None and is_a_type(attrib_type):
```

Comment:
This assumes that the `.type` attribute always exists; is that true for every attrs version we support?

Reply:
I meant to keep the `getattr`, thanks